safe-global/web-core
# fix: Add fallback for empty data in ApprovalEditor

## What it solves

Resolves #1890 

## How this PR fixes it

- Adds an empty data fallback when parsing transactions for the `ApprovalEditor`
- Adds an `ErrorBoundary` around `ApprovalEditor`

## How to test it

1. Open the Safe
2. Create and queue a transaction as described in https://github.com/safe-global/web-core/pull/1842#issuecomment-1514661723
3. Try to execute the transaction
4. Observe the app not crashing

Test creating batched NFT transfers, multisend from safe apps, batch execute from queue

## Screenshots

## Checklist
